### ISSUE TITLE
Request: Support for multiple cameras 

### ISSUE THREAD

**Issue by TheBasedDoge** [kurim/p1stream-ha#2]
I have this working on my P1S, I can access the stream and added an mjpeg card.

But, it doesn't appear to have support for multiple devices. I would like to add my P1P also.

**Comment by kurim** [comment]
Through interface no, but you can add it manual by using the ingress interface:

![image](https://github.com/kurim/p1stream-ha/assets/1880220/410e823d-82d5-4d99-a500-669f98ecbbf5)

just add a second:
```yaml
p1p: 
      - exec:python3 ./p1.py -a codehere -i iphere
```
But this might be redone everytime you restart HA+Container

**Comment by TheBasedDoge** [comment]
Yeah that worked. is there a way to stop it from needed to be redone each reboot?

**Comment by kurim** [comment]
Good question, in the current configuration of the addon no, but i think it might not be impossible

**Comment by TheBasedDoge** [comment]
Would it be possible to look into this as a feature request? It keeps resetting itself back to default, but when it works its great to monitor both my printers at once. Thanks in advance

**Comment by kurim** [comment]
Currently i'm not sure how to pass the settings from /data/config.json to go2rtc.yaml
